hyppo/independence/hhg.py

```diff
@@ -44,6 +53,10 @@ class HHG(IndependenceTest):
         where ``x`` is the data matrix for which pairwise distances are
         calculated and ``**kwargs`` are extra arguements to send to your custom
         function.
+
+    fast : boolean, default: False
+        Used to opt into fast version of test if desired.
```

Comment:
The description should be similar to other hyppo tests

Reply:
Editted.